server/chat-commands/info.ts

```diff
@@ -2440,6 +2440,12 @@ export const commands: ChatCommands = {
 		if (!/^https?:\/\//.test(link)) link = `https://${link}`;
 		link = encodeURI(link);
 		if (!room.pendingApprovals) room.pendingApprovals = new Map();
+		if (!room.settings.showThrottle) room.settings.showThrottle = 0;
+		if (!room.lastShowRequest) room.lastShowRequest = {};
+		if (room.lastShowRequest[user.id] && Date.now() - room.lastShowRequest[user.id] <= room.settings.showThrottle) {
+			return this.errorReply(`Your request was not submitted because you've been sending them too quickly.`);
```

Comment:
"You" is not really accurate. Is it supposed to be a global timer and not a per-user timer?

Reply:
It's supposed to be per-user, yes.

Reply:
Your current implementation is a global timer, in case that was unclear.